README.md

```diff
@@ -362,7 +362,8 @@ using LoggingExtras
 using HTTP
 
 function not_HTTP_message_filter(log)
-    log._module != HTTP
+    # HTTP.jl utilizes internal modules so call parentmodule(...)
+    parentmodule(log._module) !== HTTP
```

Comment:
```suggestion
    log.module !== HTTP && parentmodule(log._module) !== HTTP
```

Reply:
`.module` -> `._module`. Also, `parentmodule(HTTP)` is `HTTP` I think?